CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.20-dev0
```

Comment:
let's make this a release (no `-dev0` prefix)

Reply:
Done!